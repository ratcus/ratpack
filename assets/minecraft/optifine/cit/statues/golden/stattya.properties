items=nether_star
model=stattya
nbt.display.Name=\u0417\u043e\u043b\u043e\u0442\u0430\u044f \u0441\u0442\u0430\u0442\u0443\u044f