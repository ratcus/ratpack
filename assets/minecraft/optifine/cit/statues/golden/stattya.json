{
	"credit": "Made with Blockbench",
	"textures": {
		"2": "./gold_block",
		"3": "./stone",
		"4": "./polished_andesite"
	},
	"elements": [
		{
			"from": [4, 0, 4],
			"to": [12, 2, 12],
			"faces": {
				"north": {"uv": [0, 0, 8, 2], "texture": "#4"},
				"east": {"uv": [0, 0, 8, 2], "texture": "#4"},
				"south": {"uv": [0, 0, 8, 2], "texture": "#4"},
				"west": {"uv": [0, 0, 8, 2], "texture": "#4"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#4"},
				"down": {"uv": [0, 0, 8, 8], "texture": "#4"}
			}
		},
		{
			"from": [4, 4, 4],
			"to": [12, 5, 12],
			"faces": {
				"north": {"uv": [0, 0, 8, 1], "texture": "#4"},
				"east": {"uv": [0, 0, 8, 1], "texture": "#4"},
				"south": {"uv": [0, 0, 8, 1], "texture": "#4"},
				"west": {"uv": [0, 0, 8, 1], "texture": "#4"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#4"},
				"down": {"uv": [0, 0, 8, 8], "texture": "#4"}
			}
		},
		{
			"from": [5, 2, 5],
			"to": [11, 4, 11],
			"faces": {
				"north": {"uv": [0, 0, 6, 2], "texture": "#3"},
				"east": {"uv": [0, 0, 6, 2], "texture": "#3"},
				"south": {"uv": [0, 0, 6, 2], "texture": "#3"},
				"west": {"uv": [0, 0, 6, 2], "texture": "#3"},
				"up": {"uv": [0, 0, 6, 6], "texture": "#3"},
				"down": {"uv": [0, 0, 6, 6], "texture": "#3"}
			}
		},
		{
			"from": [8.1, 5, 6.85],
			"to": [10.2, 9.9, 9.45],
			"rotation": {"angle": 0, "axis": "y", "origin": [0.2, 0, 0]},
			"faces": {
				"north": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"east": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"south": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"west": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"down": {"uv": [0, 0, 16, 16], "texture": "#2"}
			}
		},
		{
			"from": [5.8, 5, 6.95],
			"to": [7.9, 9.9, 9.45],
			"rotation": {"angle": 0, "axis": "y", "origin": [0.2, 0, 0]},
			"faces": {
				"north": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"east": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"south": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"west": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"down": {"uv": [0, 0, 16, 16], "texture": "#2"}
			}
		},
		{
			"from": [10.2, 10.3, 6.85],
			"to": [12.25, 15.5, 9.45],
			"rotation": {"angle": 0, "axis": "y", "origin": [0.2, 0, 0]},
			"faces": {
				"north": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"east": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"south": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"west": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"down": {"uv": [0, 0, 16, 16], "texture": "#2"}
			}
		},
		{
			"from": [3.8, 10.4, 6.85],
			"to": [5.85, 15.5, 9.45],
			"rotation": {"angle": 0, "axis": "y", "origin": [0.2, 0, 0]},
			"faces": {
				"north": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"east": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"south": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"west": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"down": {"uv": [0, 0, 16, 16], "texture": "#2"}
			}
		},
		{
			"from": [5.8, 9.9, 6.65],
			"to": [10.2, 15.5, 9.65],
			"rotation": {"angle": 0, "axis": "y", "origin": [0.2, 0, 0]},
			"faces": {
				"north": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"east": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"south": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"west": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"down": {"uv": [0, 0, 16, 16], "texture": "#2"}
			}
		},
		{
			"from": [5.8, 15.4, 6.05],
			"to": [10.3, 19.8, 10.25],
			"rotation": {"angle": 0, "axis": "x", "origin": [0.2, 0, 0]},
			"faces": {
				"north": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"east": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"south": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"west": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"up": {"uv": [0, 0, 16, 16], "texture": "#2"},
				"down": {"uv": [0, 0, 16, 16], "texture": "#2"}
			}
		}
	],
	"display": {
		"thirdperson_righthand": {
			"rotation": [73, 0, 0],
			"translation": [0, 0.75, 3],
			"scale": [0.56, 0.56, 0.56]
		},
		"thirdperson_lefthand": {
			"rotation": [73, 0, 0],
			"translation": [0, 0.75, 3],
			"scale": [0.56, 0.56, 0.56]
		},
		"firstperson_righthand": {
			"translation": [0, 3.5, 3],
			"scale": [0.56, 0.56, 0.56]
		},
		"firstperson_lefthand": {
			"translation": [0, 3.5, 3],
			"scale": [0.56, 0.56, 0.56]
		},
		"ground": {
			"translation": [0, 3.25, 0]
		},
		"gui": {
			"rotation": [0, 41, 0],
			"translation": [0, -1.25, 0],
			"scale": [0.76, 0.76, 0.76]
		},
		"fixed": {
			"rotation": [-90, 0, 0],
			"translation": [0, 0.25, -19.5],
			"scale": [2.58, 2.58, 2.58]
		}
	},
	"groups": [
		{
			"name": "group",
			"origin": [8, 8, 8],
			"color": 0,
			"children": [0, 1, 2]
		},
		{
			"name": "group",
			"origin": [8, 8, 8],
			"color": 0,
			"children": [3, 4, 5, 6, 7, 8]
		}
	]
}